### ISSUE TITLE
Prisma Migrate just crashed

### ISSUE THREAD

**Issue by agryaznov** [prisma/migrate#639]
Hi Prisma Team! Prisma Migrate just crashed. 

## Versions
      
| Name        | Version            |
|-------------|--------------------|
| Platform    | debian-openssl-1.1.x| 
| Node        | v14.15.0           | 
| Prisma CLI  | 2.10.0             | 
| Binary      | af1ae11a423edfb5d24092a85be11fa77c5e499c| 

## Error
```
Error: Error in migration engine.
Reason: [migration-engine/connectors/sql-migration-connector/src/sql_renderer/postgres_renderer.rs:213:26] AlterColumn on unknown column

Please create an issue in the migrate repo with
your `schema.prisma` and the prisma command you tried to use 🙏:
https://github.com/prisma/migrate/issues/new

```

**Comment by tomhoule** [comment]
Thanks for reporting this. This is a bug that was fixed in 2.10.1, can you try with that version and confirm that it doesn't happen for you anymore?

**Comment by nonissue** [comment]
Had a similar issue with `2.10.0`, was resolved by updating to `2.10.2`

**Comment by tomhoule** [comment]
Glad the fix worked for everyone, we can close this issue then :)
